common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -0,0 +1,1319 @@
+                        '%(earned)s/%(possible)s point (ungraded)', '%(earned)s/%(possible)s points (ungraded)',
+                        possible
+                    );
+                }
+                progress = interpolate(progressTemplate, {
+                    earned: earned,
+                    possible: possible
+                }, true);
+            }
+
+            // Render 'x point(s) possible' if student has not yet attempted question
+            // Status is set to none when a user has a score of 0, and 0 when the problem has a weight of 0.
+            if (status === 'none' || status === 0) {
+                if ((detail !== null && detail !== undefined) && (jQuery.type(detail) === 'string') &&
+                    detail.indexOf('/') > 0) {
+                    a = detail.split('/');
+                    possible = parseFloat(a[1]);
+                } else {
+                    possible = 0;
+                }
+                if (graded === 'True' && possible !== 0) {
+                    progressTemplate = ngettext(
+                        // Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).;
+                        '%(num_points)s point possible (graded)', '%(num_points)s points possible (graded)',
+                        possible
+                    );
+                } else {
+                    progressTemplate = ngettext(
+                        // Translators: %(num_points)s is the number of points possible (examples: 1, 3, 10).;
+                        '%(num_points)s point possible (ungraded)', '%(num_points)s points possible (ungraded)',
+                        possible
+                    );
+                }
+                progress = interpolate(progressTemplate, {num_points: possible}, true);
+            }
+            return this.$('.problem-progress').text(progress);
+        };
+
+        Problem.prototype.updateProgress = function(response) {
+            if (response.progress_changed) {
+                this.el.data('progress_status', response.progress_status);
+                this.el.data('progress_detail', response.progress_detail);
+                this.el.trigger('progressChanged');
+            }
+            return this.renderProgressState();
+        };
+
+        Problem.prototype.forceUpdate = function(response) {
+            this.el.data('progress_status', response.progress_status);
+            this.el.data('progress_detail', response.progress_detail);
+            this.el.trigger('progressChanged');
+            return this.renderProgressState();
+        };
+
+        Problem.prototype.queueing = function(focusCallback) {
+            var that = this;
+            this.queued_items = this.$('.xqueue');
+            this.num_queued_items = this.queued_items.length;
```

Comment:
I only changed the things causing eslint violations. For some reason, this one wasn't flagged.